testnets/nobletestnet/assetlist.json

```diff
@@ -81,48 +81,48 @@
       ]
     },
     {
-      "description": "USDLR is a fiat-backed stablecoin issued by Stable. Stable pays DeFi protocols who distribute USDLR.",
+      "description": "Ondo US Dollar Yield",
       "denom_units": [
         {
-          "denom": "uusdlr",
+          "denom": "ausdy",
           "exponent": 0,
           "aliases": [
-            "microusdlr"
+            "attousdy"
           ]
         },
         {
-          "denom": "usdlr",
-          "exponent": 6
+          "denom": "usdy",
+          "exponent": 18
         }
       ],
-      "base": "uusdlr",
-      "display": "usdlr",
-      "name": "USDLR by Stable",
-      "symbol": "USDLR",
+      "base": "ausdy",
+      "display": "usdy",
+      "name": "Ondo US Dollar Yield",
+      "symbol": "USDY",
       "traces": [
         {
           "type": "test-mintage",
           "counterparty": {
-            "chain_name": "noble",
-            "base_denom": "uusdlr"
+            "chain_name": "nobletestnet",
```

Comment:
this is a self-referential pointer, which doesn't make sense. It should point to the mainnet version, but if it doesn't exist yet, then just skip the trace.